### PR TITLE
chore: Show schema as a tab instead of the right pane

### LINKED ISSUE #30873: Move Schema to the Query Debugger
Screenshot 2024-02-02 at 1 28 57 PM

### PR DESCRIPTION
**Pull request by hetunandu**
## Description

Moves the Schema in Query Pane from the right side to the tab on the bottom. This is done to eventually remove the right side pane for Side by Side layout

#### PR fixes following issue(s)
Fixes #30873

#### Media

https://github.com/appsmithorg/appsmith/assets/12022471/28c5086a-1967-4a72-8dc5-0717751198cf

#### Type of change

- New feature (non-breaking change which adds functionality)

## Testing

- [ x ] Manual
- [ x ] Jest
- [ x ] Cypress

## Summary by CodeRabbit

- **New Features**
	- Introduced a refresh button in the datasource structure container for easy refreshing.
	- Added a new state to handle scenarios with no columns to display, enhancing user feedback.
	- Implemented conditional rendering of components based on new props and states, improving UI responsiveness and interaction.
- **Enhancements**
	- Updated placeholder texts and messages for a clearer user experience.
	- Improved the layout and styling of various components for better visual appeal and usability.
- **Refactor**
	- Streamlined the rendering logic and state management in several components, leading to a cleaner codebase.
	- Removed unused imports and functions, optimizing component performance.
- **Bug Fixes**
	- Fixed issues in test suites related to table schema handling and selection.
- **UI Changes**
	- Adjusted styling properties for better alignment, display, and gap spacing in the datasource structure search container.

### CODE REVIEW

File: app/client/src/components/editorComponents/Debugger/Schema.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import { Flex } from "design-system";
```
Comment (hetunandu) [inline app/client/src/components/editorComponents/Debugger/Schema.tsx:1]: @coderabitai can you write a unit test for this component

Comment (hetunandu) [reply app/client/src/components/editorComponents/Debugger/Schema.tsx:1]: @coderabbitai can you write a unit test for this component

File: app/client/src/pages/Editor/QueryEditor/QueryResponseTab.tsx [hunk 2]
```diff
@@ -170,6 +173,15 @@ const QueryResponseTab = (props: Props) => {
     }
   }
 
+  if (isRunning) {
+    return (
+      <ActionExecutionInProgressView
+        actionType="query"
```
Comment (albinAppsmith) [inline app/client/src/pages/Editor/QueryEditor/QueryResponseTab.tsx:179]: Can this type be a enum? I see we use this in two different files.

**Review by albinAppsmith — APPROVED** [review]

**Pull request merged**